src/lib/viewers/image/ImageViewer.js

```diff
@@ -178,6 +179,10 @@ class ImageViewer extends ImageBaseViewer {
         let height;
         let width;
         const isRotated = this.isRotated();
+        const viewport = {
```

Comment:
Moved this const up to the top because it's being shared by both the if and else blocks.